common/lib/xmodule/xmodule/tests/test_video.py

```diff
@@ -522,21 +536,75 @@ def test_import_with_float_times(self):
             'data': ''
         })
 
+    @patch('xmodule.video_module.video_module.edxval_api')
```

Comment:
As we discussed, I want to note my concern of integration testing with only mocked edx-val apis.  We have been bitten in the past where the edx-val API had changed and had broken the edx-platform usage of it, because the only integration test for it was using a mocked API.

I strongly suggest adding non-mocked integration tests in lms/djangoapps/courseware/tests/test_video_mongo.py where tests using edx-val are located - either in this PR or in a subsequent PR.